excel/src/main/java/org/apache/metamodel/excel/ExcelConfiguration.java

```diff
@@ -38,25 +38,38 @@
 	public static final int NO_COLUMN_NAME_LINE = 0;
 	public static final int DEFAULT_COLUMN_NAME_LINE = 1;
 
+	private final int getNumberOfLinesToScan;
```

Comment:
Can you rename `getNumberOfLinesToScan` parameter to `numberOfLinesToScan`?

Reply:
Done with next commit.